### PR TITLE
RN upgrade

### PR DESCRIPTION
**Pull request by swansontec**
### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [x] Yes
- [ ] No

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [x] Tested on iOS device
- [x] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206480248002623

### CODE REVIEW

**Review by samholmes — COMMENTED** [review]
Still have to finish review on the main commit. So leaving these comments here for now.

File: android/app/src/main/java/co/edgesecure/app/MainApplication.kt
[the remark this answers is not in the document]
Comment (samholmes) [reply android/app/src/main/java/co/edgesecure/app/MainApplication.kt:1]: It was using a menu option in the Android Studio IDE

File: package.json
[the remark this answers is not in the document]
Comment (samholmes) [reply package.json:1]: To make sure the diff is clearer from a new RN 73 init project to this one

**Review by samholmes — APPROVED** [review]

File: src/__tests__/components/__snapshots__/Buttons.test.tsx.snap
[the remark this answers is not in the document]
Comment (samholmes) [reply src/__tests__/components/__snapshots__/Buttons.test.tsx.snap:1]: Or, add the explanation that this commit runs the command `yarn test -u` following the previous commit. Mention the commit message title of the previous commit for reference.

File: ios/Podfile.lock [hunk 1]
```diff
@@ -604,35 +604,10 @@ PODS:
...
-    - React-jsiexecutor
-    - React-jsinspector
-    - React-RCTActionSheet
-    - React-RCTAnimation
-    - React-RCTBlob
-    - React-RCTImage
-    - React-RCTLinking
-    - React-RCTNetwork
-    - React-RCTSettings
-    - React-RCTText
```
[the remark this answers is not in the document]
Comment (samholmes) [reply ios/Podfile.lock]: answer: 🤷‍♂️

File: src/components/charts/SwipeChart.tsx
[the remark this answers is not in the document]
Comment (swansontec) [reply src/components/charts/SwipeChart.tsx:1]: The old setup was causing TypeScript errors post-upgrade. Fixing the refs made the types line up.

**Pull request merged**